dynamo/preprocessing/preprocessor_utils.py

```diff
@@ -1130,19 +1132,21 @@ def filter_cells_by_outliers(
         adata, layer_keys_used_for_filtering, predefined_range_dict, shared_count
     )
 
-    if filter_bool is None:
-        filter_bool = detected_bool
-    else:
-        filter_bool = np.array(filter_bool) & detected_bool
+    if max_pmito_s is not None:
+        detected_bool = detected_bool & (adata.obs["pMito"] < max_pmito_s)
+        main_info(
+            "filtered out %d cells by %f%% of mitochondrial genes for a cell."
+            % (adata.n_obs - (adata.obs["pMito"] < max_pmito_s).sum(), max_pmito_s),
```

Comment:
(adata.n_obs - (adata.obs["pMito"] < max_pmito_s).sum() this is the number of remaining cells not the cells that are removed

Reply:
Thank you for correcting. 
Fixed.